### PR TITLE
install: support of tarantool non-static build

### PR DESCRIPTION
**Pull request by psergee**
Added new option --dynamic for tt install tarantool command. This options enables dynamic linking for building tarantool executable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,8 +30,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - A configurable variable `cluster_cookie` for `tt create cartridge` template.
 - ``tt build`` accepts application name for building.
-- Creating wal, vinyl and memtx directories for `tt pack`. If these directories 
-  are not located in the same directory in the environment for packing, the result package 
+- Creating wal, vinyl and memtx directories for `tt pack`. If these directories
```
Comment (better0fdead) [inline CHANGELOG.md:33]: This changes doesn't belong to the commit.

**Review by better0fdead — COMMENTED** [review]
LGTM, just small comment.

File: CHANGELOG.md (hunk 1 above)
Comment (psergee) [reply CHANGELOG.md:33]: Right. Got into it again.

**Review by better0fdead — APPROVED** [review]
